generators/newcomponent/index.js

```diff
@@ -0,0 +1,118 @@
+const Generator = require('yeoman-generator');
```

Comment:
Do you think it would make sense to rename this sub-generator to something like "newcomponentusage" to avoid misunderstandings?
I don't have a strong opinion on this one. It's up to you.